src/core/EnvelopeAndLfoParameters.cpp

```diff
@@ -274,7 +272,7 @@ inline void EnvelopeAndLfoParameters::fillLfoLevel( float * _buf,
 	}
 
 	fpp_t offset = 0;
-	const float lafI = 1.0f / qMax( minimumFrames, m_lfoAttackFrames );
+	const float lafI = 1.0f / m_lfoAttackFrames;
```

Comment:
I'm curious for  your rationale behind removing the minimum here. Can you guarantee that m_lofAttackFrames will never be 0?

Reply:
> Can you guarantee that m_lofAttackFrames will never be 0?

He can't. I believe I added that statement early after stable-1.2 was branched off master. One of the things that most likely will break by this change is the fpe debug option (WANT_DEBUG_FPE). @musikBear What was the reason behind this change?

Reply:
> What was the reason behind this change?

I have *N O T* made these changes!
I do not have the faintest idea, why they are in this commit!
Could that happen if i updated one file through GIT?
Bottom-line : The only code i have added in EnvelopeAndLfoParameters.cpp is for the Envelopes, nothing else has been touched by me
My code:
```
//  m_predelayModel( val, min, max, incr,  this, tr( ">info<" ) )   
	m_predelayModel( 0.0, 0.0, 2.0, 0.001, this, tr( "Env pre-delay" ) ),
	m_attackModel( 0.025, 0.0, 2.0, 0.001, this, tr( "Env attack" ) ),
	m_holdModel( 0.0, 0.0, 2.0, 0.001,     this, tr( "Env hold" ) ),
	m_decayModel( 0.6, 0.0, 2.0, 0.001,    this, tr( "Env decay" ) ),
	m_sustainModel( 0.0, 0.0, 1.0, 0.001,  this, tr( "Env sustain" ) ),
	m_releaseModel( 0.2, 0.0, 2.0, 0.001,  this, tr( "Env release" ) ),
        m_amountModel( 1.0, -1.0, 1.0, 0.005,  this, tr( "Env mod amount" ) ), //AMP 
```
And ONLY those lines are mine.
This is my fault, no question, but HOW i have gotten an unknown file 'merged'(?) into my commit, i do not know. I used the 'add-file-through-upload'.
I definitively expected that my 'added' file would **replace** the current!
I does **NOT**?!?

Reply:
This must be something with your git workflow then... I would expect this to show up as a merge conflict under a typical fork -> commit -> PR scenario.

My guess here is that there are changes in `LMMS/lmms/master` that you don't have yet. You may have to manually update your code with the changes in master.

Reply:
>  I definitively expected that my 'added' file would **replace** the current!

You shouldn't add files. You should push commits. Your master branch is borked and needs to be reset to the lmms master branch.


>  I used the 'add-file-through-upload'.

What's that? Are you uploading files via the GitHub interface? Have you followed any tutorial to learn how to work with git/GitHub?